src/test/test.js

```diff
@@ -14,6 +14,7 @@ describe('Floccus', function() {
   this.slow(20000) // 20s is slow
   before(async function() {
     const background = await browser.runtime.getBackgroundPage()
+    window.authManager = background.authManager
```

Comment:
I wonder if we can put that into the AuthSession directly :thinking:

Reply:
well you could, but:
It would require an async AuthSession.initialize(), because getBackgroundPage is async and you can't await in constructor() (which is sync). 
Should i do nevertheless?

Reply:
We could call the initialize in the onSyncStart hook, I think that would be better for encapsulation :)